src/fdrs_lora.h

```diff
@@ -286,6 +289,52 @@ void transmitLoRa(uint16_t *destMac, SystemPacket *packet, uint8_t len)
     return;
 }
 
+// Synchronously Transmits LoRa data and, if configured, listens for ACKs
+// Return type is crcResult struct - CRC_OK, CRC_BAD, CRC_NULL.  CRC_NULL used for non-ack data
+crcResult transmitLoRaSync(uint16_t *destMac, DataReading *packet, uint8_t len)
+{
+    crcResult crcReturned = CRC_NULL;
+    // Do we check to see if another Async operation is in progress first???
+    // If Yes, what do we do if another operation is in progress?  Return and rely on retransmit retry?
```

Comment:
I think that if an operation is in progress then we should either:
- Wait until it finishes and queue it after. I think this is probably messy and non-ideal.
- Fail. It would be nice to return -1 for 'busy', then maybe the user could do `while(sendFDRS() = -1){delay(100);}` to retry. That's a bit extra though. In the end, the user should probably just be careful using both types in the same sketch. I guess there's also the possibility of a receive op happening.

I would say I'm fine with just returning false (from `sendFDRS()`) if there's an op in progress.